src/app/generate-blurb/page.tsx

```diff
@@ -1,7 +1,7 @@
-import TextGenerationForm from "@/components/TextGenerationForm";
+import Sidebar from "@/components/Sidebar";
 
 const GenerateBlurb = () => {
-  return <TextGenerationForm />;
+  return <Sidebar img="hello" />;
```

Comment:
```html
<Sidebar>
 <TextGenerationForm/>
</Sidebar>
```
Now I think about it, this is more like a layout thing, so we should probably put it in the `layout.tsx` file